src/views/tables/PositionsTable/PositionsTriggersCell.tsx

```diff
@@ -0,0 +1,147 @@
+import styled, { type AnyStyledComponent, css } from 'styled-components';
+
+import { AbacusOrderTimeInForces, AbacusOrderTypes, Nullable } from '@/constants/abacus';
+import { ButtonAction, ButtonSize, ButtonType } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+import { TimeInForceOptions } from '@/constants/trade';
+
+import { useStringGetter } from '@/hooks';
+
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { Button } from '@/components/Button';
+import { Icon, IconName } from '@/components/Icon';
+import { Output, OutputType } from '@/components/Output';
+
+import { isMarketOrderType } from '@/lib/orders';
+
+export type PositionTableConditionalOrder = {
+  price: number;
+  size: number;
+  triggerPrice: Nullable<number>;
+  timeInForce: Nullable<AbacusOrderTimeInForces>;
+  type: AbacusOrderTypes;
+};
+
+type ElementProps = {
+  stopLossOrders: PositionTableConditionalOrder[];
+  takeProfitOrders: PositionTableConditionalOrder[];
+  positionSize?: number | null;
+  isDisabled?: boolean;
+};
+
+export const PositionsTriggersCell = ({
+  stopLossOrders,
+  takeProfitOrders,
+  positionSize,
+  isDisabled, // TODO: CT-656 Disable onViewOrdersClick behavior when isDisabled
+}: ElementProps) => {
+  const stringGetter = useStringGetter();
+
+  const onViewOrdersClick = () => {
+    // TODO: CT-655
+  };
+
+  const renderOutput = ({
+    label,
+    orders,
+  }: {
+    label: string;
+    orders: PositionTableConditionalOrder[];
+  }) => {
+    const triggerLabel = (warning: boolean = false) => (
+      <Styled.Label warning={warning}>{label}</Styled.Label>
+    );
+    const viewOrdersButton = (
+      <Styled.Button
+        type={ButtonType.Link}
+        action={ButtonAction.Navigation}
+        size={ButtonSize.XSmall}
+        onClick={onViewOrdersClick}
+      >
+        {stringGetter({ key: STRING_KEYS.VIEW_ORDERS })}
+        {<Icon iconName={IconName.Arrow} />}
+      </Styled.Button>
+    );
+
+    if (orders.length === 0) {
+      return (
+        <>
+          {triggerLabel()} <Styled.Output type={OutputType.Fiat} value={null} />
+        </>
+      );
+    } else if (orders.length === 1) {
+      const { price, size, triggerPrice, timeInForce, type } = orders[0];
+
+      const isPartial = !!(positionSize && Math.abs(size) < Math.abs(positionSize));
+      const shouldRenderValue =
+        timeInForce?.name === TimeInForceOptions.IOC &&
+        (isMarketOrderType(type) || price === triggerPrice);
+
+      return shouldRenderValue ? (
```

Comment:
I think you can just do `if (shouldRenderValue) { ... }` and remove the `else` since the return will always be 

```
<>
  {triggerLabel()}
  {viewOrdersButton}
</>
```